hummingbot/connector/exchange/btc_markets/btc_markets_utils.py

```diff
@@ -0,0 +1,98 @@
+import math
+from typing import Dict, List
+from dateutil.parser import parse as dateparse
+
+from hummingbot.core.utils.tracking_nonce import get_tracking_nonce, get_tracking_nonce_low_res
+from . import btc_markets_constants as Constants
+
+from hummingbot.client.config.config_var import ConfigVar
+from hummingbot.client.config.config_methods import using_exchange
+
+
+CENTRALIZED = True
+
+EXAMPLE_PAIR = "XRP-AUD"
+
+# Using AUD Market Pair rates, but need to figure out how to determine 30 day AUD vol for rolling
+# https://www.btcmarkets.net/fees
+DEFAULT_FEES = [0.85, 0.85]
+
+HBOT_BROKER_ID = "HBOT-"
```

Comment:
Is this the ID assigned by the exchange to Hummingbot? If not there is no point on using a broker id.

Reply:
Based on crypto.com connector - https://github.com/hummingbot/hummingbot/blob/master/hummingbot/connector/exchange/crypto_com/crypto_com_utils.py#L17